doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -154,9 +154,13 @@ bash configure --with-freemarker-jar=/root/freemarker.jar --with-boot-jdk=/usr/l
 
 :warning: The path in the example --with-boot-jdk= option is appropriate for the Docker installation. If not using the Docker environment, set the path appropriate for your setup, such as "/<my_home_dir>/bootjdk8" as setup in the previous instructions.
 
-:pencil: **Non-compressed references support:** If you require a heap size greater than 57GB, enable a noncompressedrefs build with the `--with-noncompressedrefs` option during this step.
+:pencil: **Mixed and compressed references support:** A 64-bit build can support mixed references which includes both compressed and non-compressed references, compressed references only, or non-compressed references only. When using mixed references the [compressed state](https://www.eclipse.org/openj9/docs/gc_overview/#compressed-references) is selected at runtime. Mixed references is the default to build when no options are specified. _Note that `--with-cmake` (which is the default) must be used when building mixed references._ `configure` options include:
```

Comment:
I think it would be clearer to invert the comment about using cmake:

> Note that support for mixed references is not compatible with `--with-cmake=no`.

Reply:
Updated.

Reply:
I'm thinking to re-write that as: Note that `--with-cmake=no` cannot be used to build mixed references.

Reply:
Pushed that change.

Reply:
I think I figured out how this sentence is *supposed* to be parsed:

> A 64-bit build can support mixed references which includes both compressed and non-compressed references, compressed references only, or non-compressed references only.

Unfortunately, it's not clear that compressed-only and non-compressed-only are separate from mixed support. Perhaps it would help to reorder the clauses as they're listed below

> A 64-bit build can be created to support:
> * compressed references (only)
> * non-compressed references (only)
> * both compressed and non-compressed references (selected as the VM is created)

Reply:
Pushed a change.

Reply:
and tweaked it.

Reply:
Much improved: thanks.